sproul-club/sc-frontend
# Timothy/scf 67 fixing new club

[scf-67:](https://scfrontend.atlassian.net/browse/SCF-67?atlOrigin=eyJpIjoiZWM2YWI4ZDA0NmFiNDQ4MWE0OGVkM2YwZWE3YmYwMjIiLCJwIjoiaiJ9)

Steps to test:
1. Use the +New button 
2. Click on the clubs in the board to make sure that they're still functioning properly 

Screenshots:
<img width="1440" alt="Screen Shot 2021-11-05 at 1 51 16 PM" src="https://user-images.githubusercontent.com/70298071/140576917-eade9590-9016-4fbe-8be4-c2910161b096.png">
